.github/workflows/test.yml

```diff
@@ -0,0 +1,14 @@
+on: push
```

Comment:
Shouldn't this be `on: [pull_request, push]`

Reply:
yes, I have updated the PR.